frontend/src/components/LeetCodeInstructions.tsx

```
@@ -0,0 +1,78 @@
+import styles from "@/styles/Instructions.module.css";
+
+export default function LeetCodeInstructions() {
+    const [isVisible, setIsVisible] = useState(false);
+
+    const toggleVisibility = () => {
+        setIsVisible(!isVisible);
+    };
+
+    return (
+        <div className="guideContainer">
+            <div className={styles.title} onClick={toggleVisibility}>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)